File: 1-1.rb
#!/usr/bin/env ruby

readings = File.read('1.input').lines.map(&:to_i)

increments = 0
last = readings[0]

1.upto(readings.size - 1) do |i|
  increments += 1 if readings[i] > last
  last = readings[i]
end

print increments, "\n"

File: 1-1.py
#!/usr/bin/env python

readings = [int(row) for row in open('1.input').readlines()]

increments = 0
last = readings[0]

for i in range(1, len(readings)):
    if readings[i] > last:
        increments += 1
    last = readings[i]

print(increments)
